src/pages/filter/ui/selected-filter-card.tsx

```diff
@@ -40,11 +50,11 @@ export const SelectedFilterCard = observer(
 
         {isOpen && (
           <div>
-            {filterKeys.map(filterKey => (
-              <div key={filterKey} className="flex items-center pl-6 py-4">
-                <Checkbox checked onChange={() => onRemove(filterKey)} />
+            {filtersEntries.map(filter => (
```

Comment:
Try something that makes sense 
`{filtersEntries.map(([filterName, filterExpression])=> (`
would be more self-describing